src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -312,6 +314,11 @@ public void UndoApprenticeshipUpdate(Party party, UserInfo userInfo)
             return newPriceHistory.ToList();
         }
 
+        public void EmitOverlappingTrainingDateResolvedNotificationEvent(long apprenticeshipId, long cohortId)
+        {
+            Publish(() => new OverlappingTrainingDateResolvedEvent(apprenticeshipId, cohortId));
+        }
```

Comment:
This does not need to be public as only called from within this class.

The separate method does not add anything I would simply replace line 74 with line 319

Reply:
Done